google/cloud/spanner/internal/database_admin_logging_test.cc

```diff
@@ -89,13 +96,20 @@ TEST_F(DatabaseAdminLoggingTest, GetDatabaseDdl) {
 }
 
 TEST_F(DatabaseAdminLoggingTest, UpdateDatabase) {
-  EXPECT_CALL(*mock_, UpdateDatabase).WillOnce(Return(TransientError()));
+  EXPECT_CALL(*mock_, AsyncUpdateDatabaseDdl)
+      .WillOnce([](CompletionQueue&, std::unique_ptr<grpc::ClientContext>,
+                   gcsa::UpdateDatabaseDdlRequest const&) {
+        return make_ready_future(
+            StatusOr<google::longrunning::Operation>(TransientError()));
+      });
 
   DatabaseAdminLogging stub(mock_, TracingOptions{});
 
-  grpc::ClientContext context;
-  auto status = stub.UpdateDatabase(context, gcsa::UpdateDatabaseDdlRequest{});
-  EXPECT_EQ(TransientError(), status.status());
+  CompletionQueue cq;
+  std::unique_ptr<grpc::ClientContext> context(new grpc::ClientContext);
```

Comment:
No biggie, but consider `absl::make_unique`.

Reply:
Yeah ... I didn't want to introduce a new dependency here and now, but I'll happily do so in a followup.